### ISSUE TITLE
Identify data requirements

### ISSUE THREAD

**Issue by ianhinder** [ScottishCovidResponse/SCRCIssueTracking#394]
Make a list of the different data inputs that are needed, and arrange for these to be provided.

**Comment by ianhinder** [comment]
I've made a rough draft of the data we need based on what the code currently uses, and have asked Glenn and Chris to complete/correct it.

**Comment by ianhinder** [comment]
Chris has made a list of the data requirements:

1) For the geographical mixing matrix we are using the flow data from the census. This will be at two scales: MSOA level and OA level.

2) Demographic infomation at MSOA level and OA level (the most important information is population size and geographical location of areas, e.g. boundary data, so that might be enough).

3) Weekly hospitalised case data for each healthboard.

4) Weekly hospitalised death data (nationwide is ok).

5) Distributions of cases for different factors (age, sex, any other factors).

6) Equivalent distributions for the population as a whole.

7) Distribution in house sizes and ages within houses (availible from census data I think).

8) Contact matrices for differnt age groups available here:
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005697

**Comment by ianhinder** [comment]
I have uploaded versions of the input data files that the code currently uses to the temporary_data repository. There are README.md files accompanying each file, based on a description emailed by Chris. The datasets are:
- [Names of health board regions and the codes they used ](https://github.com/ScottishCovidResponse/temporary_data/tree/master/coronapmcmc/geographic/regionlabels/scotland/data)
- [Age mixing matrix](https://github.com/ScottishCovidResponse/temporary_data/tree/master/coronapmcmc/human/age-mixing-matrix/data)
- [Areas over which disease spread is modelled](https://github.com/ScottishCovidResponse/temporary_data/tree/master/coronapmcmc/human/demographics/scotland/by-msoa/data)
- [Geographic mixing](https://github.com/ScottishCovidResponse/temporary_data/tree/master/coronapmcmc/human/geographic-mixing-matrix/scotland/by-msoa/data)

**Comment by chrispooley** [comment]
Hi,
Not sure what to do here. Richard B is now in charge of data, and we are slowly piecing together what we need...

**Comment by ianhinder** [comment]
Fair enough. I'll leave this open just to remind us that we still need to pin things down a bit. I'll add Richard to this issue when I find out his GitHub username.
